modules/widgets/gravatar-profile.php

```diff
@@ -230,7 +230,7 @@ function enqueue_scripts() {
 
 		wp_enqueue_style(
 			'gravatar-card-services',
-			is_ssl() ? 'https://secure.gravatar.com/css/services.css' : 'http://s.gravatar.com/css/services.css',
+			is_ssl() ? 'https://secure.gravatar.com/css/services.css' : 'https://s.gravatar.com/css/services.css',
```

Comment:
Either need to leave this as a `is_ssl` conditional with the http option or just use one URL. both `secure` and `s` appear to work fine under HTTPS.